src/System.Runtime/tests/System/Reflection/ModuleTests.cs

```diff
@@ -188,7 +188,6 @@ public void ResolveType(Type t)
             {
                 new object[] { 1234 },
                 new object[] { typeof(ModuleTests).GetMethod("ResolveType").MetadataToken },
-                new object[] { typeof(ModuleTests).MetadataToken + 10000 },
```

Comment:
This appears to have been an attempt to get a bad token id.  Now that we're removing it, are we leaving a test hole that needs to be filled by something else?